### ISSUE TITLE
[NPC][Karazhan]Prince Malchezaar

### ISSUE THREAD

**Issue by Picajzla** [Atlantiss/NetherwingBugtracker#5062]
I'm gonna open a new issue as the last one got closed before i commented on it. The closed issue is "only" complaining about 9k axe hits on prince,but that's only 1 of many issues that fight has. I'd like if you could link me those "prenerf" value sources  because I'm seriously struggling to find a video where tank gets hit for 19k with ironshield pot on as it happens on this server. The following kill video from retail happened on 4/2/2007 (less than 20 days from release of TBC):  https://www.youtube.com/watch?v=wHyqB2Ruqyk where you can clearly see a 1.5k crit on cloth. Normal hits are 700. On this server clothies get hit for 9k insta. You can see the priest getting hit on 6:20
I'm not even gonna get started on the cleave in this video. None of the melees get cleaved like they get on this server and they're running all around the boss. Getting cleaved on prince on this server happens when you're behind the boss and you get 1 shot. I'm gonna shut up and never complain about Prince again  if you can counterproof me with some legit evidence that you claim to be your source. A server that has bugs and tries to fix them is a server that i can play on, but server that refuses to admit their mistakes and won't work on fixing them is not worth wasting time on.

**Comment by Mewmewww** [comment]
Personally, our group didn't have any problems with flying axe phase and it was our first prince run. Cleave didn't hit anyone behind if you positioned yourself correctly behind him at max range (we had a rogue and a enhancement shaman)

**Comment by Firroth** [comment]
You can counter cleave by standing at max range. Axes aren't that much of a problem if group expects it.

"I'm seriously struggling to find a video where tank gets hit for 19k with ironshield pot on as it happens on this server." - Maybe because your tank sucks? 19k is only possible if he gets crushed during thrash (thrash is insta 3 hits, so if 2 of those are crush for 6-8k then no wonder, he got rekt) or comboed with cleave. I'm tanking him with 14k hp and died only once to him by insta-killing me in 4 Id's.

**Comment by femdead** [comment]
if you get 2 crushes during enrage it's more like 27K than 19K and shield block doesn't have infinite charges, you've gotten lucky so far firroth so good for you i guess

**Comment by Firroth** [comment]
@femdead  
Thing is i'm not stama stacking, so I simply don't get hit by thrash at all (he can't even stacl sunder armor on me (max 2 stacks), since he misses most of the time (35-40% chance to hit me), so if he lands his thrash on me, all 3 hits from it are on roll - hit me or not with this low hit chance, so far only once all landed. Also since he has low hit chance on me, my shield block charges aren't taken in 1-2s, like it is for other tanks who prio stama. (that's my playstyle, swapping gear depending on boss, not stama stacking).

One question though: "if you get 2 crushes during enrage it's more like 27K than 19K" - what do you mean by enrage? you call p2 this way?

**Comment by Picajzla** [comment]
I guess if you expect the axes happening, they aren't much of a problem right?https://i.gyazo.com/thumb/1200/838deff1b7c03e273f1ee3b3a2d2b868-png.jpg
Unless this happens on a target that doesn't really have 9k hp. Might as well watch the video that i linked if you're saying that we're getting cleaved. Pretty sure they'd be cleaved as well running all around the boss. Not to mention the fact that priest in the video gets hit for 10x less. This fight is a complete RNG as it is now. One time you kill it with no problems, other time tank rolls over and dies, and then the next time axes just come up to you and hit you in the face with amount larger than your hp.

**Comment by Firroth** [comment]
I'm not questioning nor stating it should be that way. My healers priest and paladin never cried about it nor died to it, so I don't really knew it was this way and it had such dmg to begin with. But you're right it shouldn't hit for this much, so it should be fixed. 
As for cleave 2.0 had 360 cleaves, so be happy it isn't like that here. It's propably because of it's bounding radius or smth along those lines which make it look like 360. If your meele will stay at max range, but still in range of meele attacks they won't get cleaved I assure you. Devs should look into it anyway I guess.

**Comment by Tenoutoften** [comment]
I guess this is slightly off topic as some mobs use different cleave spells but the range of it does look strange at times. [Here] is me taking a cleave from a Lurker add when I am quite far away from the tank.
![](https://i.gyazo.com/4ee58de1af9daf76c4e417cb60e2cbeb.png)

**Comment by Tenoutoften** [comment]
I also think it is important for people to understand that this server does intentionally use custom values. Entire SSC is filled with custom health values, expect the same for Tempest Keep and other dungeons

**Comment by femdead** [comment]
@Firroth you are aware that shield block charges, sunder stacks, and crushing blows are all down to rng? No matter how you gear your avoidance is only going to be within a few points of another tank unless you're unironically gemming parry (on-use trinkets nonwithstanding but the phase lasts much longer than your cooldowns are available)

**Comment by Neophrone** [comment]
The problem is that this cleave and axes damage ARE NOT BLIZZLIKE. I know that a lot raids kill it with those custom changes. But dews of NW aiming Blizzlike quality. So fix it or prove that it's Blizzlike. I know that melee can avoid this cleave, but than tank will eat those cleaves. Second phase is completly rng. Our tank died 2 times because of trash and this cleave happened at the same time.

**Comment by Firroth** [comment]
@femdead 

![screenshot_1](https://user-images.githubusercontent.com/43505196/52954682-14909080-338b-11e9-9215-e0eb14f230fd.png)

I'm not even t4 geared except, t4 gloves, and yet I have 50% avo (dodge+parry+miss, without block). Adding buffs to it Major agi elixir, BoK, wild, heck even 20agi food buff, I'm then on 24/25% dodge (55% total avo), add to it hunter's sting +3% miss and agi totem +3-4% dodge = 61-62%.
My hp is at 15k full buff with all this. So like I wrote he has 38% chance to actually hit me, I don't even bother stoping to attack him during p2, since parry haste isn't an issue.

**Comment by Easyfraggy** [comment]
Just a slight correction your chance to be hit by a lvl 73 boss is actually 102,4% (as your avoidance is lowered slightly by his higher level). Just in case you're not aware.

**Comment by Firroth** [comment]
Yeah, I forgot about the 73lvl. Anyway Easyfraggy does alcohol reduce boss lvl too?

**Comment by Easyfraggy** [comment]
Some tips for Prince:

- Make sure your dps take it easy in phase 1 so they dont get threat capped in phase 2.
- Bloodlust in phase 2.
- Keep Thunderclap and Demo up at 100% uptime during p2. Especially thunderclap helps your Shield Block uptime tremendously. (together they are over 40% damage reduction to the boss).
- Use minor cooldown (trinkets etc) early at 1-3 stacks of sunder armor.
- Use Last Stand at 4-5 followed by immediate Shield wall when Last stand fades.
- Furthermore don't use windfury on the tank during p2. Tank can even stop melee attacks during high sunders (3-5) to avoid parryhasting the boss. This last part puts even more emphasis on dps staying low on threat in phase 1.

**Comment by Easyfraggy** [comment]
> Yeah, I forgot about the 73lvl, but still it doesn't change the fact I don't get oneshooted, like other tanks tend to get. Anyway @Easyfraggy does alcohol reduce boss lvl too?

Alcohol only visually lowers mob level - it doesn't actually lower their level =)

**Comment by Firroth** [comment]
> 
> 
> > Yeah, I forgot about the 73lvl, but still it doesn't change the fact I don't get oneshooted, like other tanks tend to get. Anyway @Easyfraggy does alcohol reduce boss lvl too?
> 
> Alcohol only visually lowers mob level - it doesn't actually lower their level =)

Really? Since f.e during lvling I used alcohol on high lvl mobs, to drop their lvl by 5-6 and from 60%+ misses it went down to like 20-30% when dual wielding (1-60 lvling), without any hit gear.

**Comment by Easyfraggy** [comment]
> > > Yeah, I forgot about the 73lvl, but still it doesn't change the fact I don't get oneshooted, like other tanks tend to get. Anyway @Easyfraggy does alcohol reduce boss lvl too?
> > 
> > 
> > Alcohol only visually lowers mob level - it doesn't actually lower their level =)
> 
> Really? Since f.e during lvling I used alcohol on high lvl mobs, to drop their lvl by 5-6 and from 60%+ misses it went down to like 20-30% when dual wielding (1-60 lvling), without any hit gear.

Should only be visual - maybe just rng?

**Comment by PolarCookie** [comment]
![triple](https://cdn.discordapp.com/attachments/335081830602637319/546799619481403392/unknown.png)
Regarding Prince one-shotting tanks, is he really supposed to triple hit them?
I know he's supposed to double hit because dual wielding, but before you scream thrash consider this
![wowscrnshot_021719_212255](https://user-images.githubusercontent.com/1158801/52962491-a86b5800-339d-11e9-9c72-56cc74d0ebf0.jpg)

@Easyfraggy I'm just curious, would a 2nd thrash have hit me there if I was a WotLK tank and could actually take 20k+ damage, i.e. 5 attacks in one instant? and just for clarification, do you have sniffs of Prince triple hitting, double thrashing and cleaving the tank in one instant?

**Comment by Firroth** [comment]
![screenshot_2](https://user-images.githubusercontent.com/43505196/52963341-cb970700-339f-11e9-8101-3a76a9260096.png)

**Comment by PolarCookie** [comment]
look closer, it's called `melee` and `melee (thrash)` in the log; the triple hit was not obviously a thrash.

**Comment by Firroth** [comment]
Just because an addon doesn't call it thrash, doesn't mean it wasn't. Addon's aren't 100% dependable in this sort of cases. Instant 3 hits in 0.01 is Thrash for me, since obv. he doesn't have attack speed 0.01

**Comment by femdead** [comment]
still not sure how this guy dropping 200 stamina for 3% extra avoidance is magically uncrushable

**Comment by Easyfraggy** [comment]
You cannot get uncrushable unless you get 102,4% avoidance+block which is very possible for paladins (holy shield) but not for warriors without shield block (only 2 charges).

**Comment by femdead** [comment]
he meant that gearing for avoidance means you're less likely to have your shield block charges fall off before you re-cast it which, yes, is a good strategy for prince, but no matter how you gear it's still possible to get bad avoidance rng especially since the enrage phase is too long to simply chain your cooldowns at this gear/dps level

so the fight is basically "hit phase 2 and pray that you don't get unlucky enough to get thrashed for full damage until you push through the phase" because the dmg potential of the boss is easily 20-30K in one swing if you lowroll

**Comment by Firroth** [comment]
If you have a chance to avoid more than 50% of all his "total" hits, then it increases your chance to not get "thrashed", and even if thrash lands in worst case scenario, then again all 3 bonus hits from it are on roll if they will hit or not (with higher chance to not hit). So even if I get thrashed, I have a chance to dodge/parry/miss the attacks from it, and it won't cummulate to 15k+ total. Shield block charge's aren't taken from you this fast too, which is a good thing. I always used 27%-30% dodge/17-18%parry/11% miss strat on this boss and it very rarely failed me.

We don't lust on p2, so I need to live good 2min+ on it. (60% trinket for 900 hp + LoH for armor+ Ironshield pot , 55% trinket from JC, 45% LS, 35% ShWall). That's how my rotation goes.

**Comment by femdead** [comment]
do you not understand the concept of rng

**Comment by Anonyxx** [comment]
Somebody actually knows how to play TBC warrior tank (not stam stacking) and you question his ability to tank... wow...

**Comment by Easyfraggy** [comment]
Chaining cooldowns is the way to get through phase 2 without dying and your dps doing as much dps as possible in this phase. There is little rng if you do this properly. Read the tips I gave earlier for warrior tanks. You shouldn't get rng'd if you do all of those things.

About the cleaves chain cleaves were how most cleaves worked in patch 2.0 where it just has a set jump distance causing you to sometimes get hit behind the target. This only happens with bad positioning e.g. tank being too close to the boss.

In patch 2.1 chain cleaves were removed and all of them turned into regular cleaves you all know from the remainder of wow history. Same will happen on NW in 2.1.

**Comment by Picajzla** [comment]
Don't know why I even bothered writing this thread and tried finding an old enough video :) Here's a meme to end this issue accordingly. https://imgflip.com/i/2u0qeo

**Comment by 5f3759** [comment]
@Easyfraggy Your arrogance and ignorance will be the death of this server.
> you wont find any videos of these
Literally the first post in this thread.

Your values are plain wrong. You claim your sniffs from MoP is more reliable than actual video evidence from pre-nerf bosses. There is no evidence good enough for you, and there's NOTHING anyone can do to convince you that you are WRONG. You're the Trump of private servers.
I give up. You can keep playing around in your sandbox, but it will never ever be blizzlike if you ignore the facts.

**Comment by Easyfraggy** [comment]
I haven't said anything about the hp/dmg values just the abilities usage fyi. 

I'm not a dev btw.

**Comment by 5f3759** [comment]
> I haven't said anything about the hp/dmg values just the abilities usage fyi.

https://github.com/Atlantiss/NetherwingBugtracker/issues/4958#issuecomment-462322960
> The axe damage is correct.

**Comment by Easyfraggy** [comment]
> > I haven't said anything about the hp/dmg values just the abilities usage fyi.
> 
> [#4958 (comment)](https://github.com/Atlantiss/NetherwingBugtracker/issues/4958#issuecomment-462322960)
> 
> > The axe damage is correct.

The axes - yes not the phase 2 damage - theres actually a bug report open on that issue where I explain a bug - wolf hasn't fixed it yet.

**Comment by Wolffenstein** [comment]
There is a bug that causes Prince to get unreasonably high damage in Phase 2 under certain circumstances. It has nothing to do with our prenerf/tuning/whatever you call it.

**Comment by gabbat** [comment]
> There is a bug that causes Prince to get unreasonably high damage in Phase 2 under certain circumstances. It has nothing to do with our prenerf/tuning/whatever you call it.

if this is known to devs, can we get a Bug report on this and an ETA on the fix for it? As of right now, some guilds disband and quit the server due to malch beeing an unbeatable RNG fest

**Comment by knasen88** [comment]
i like this one
![bild](https://user-images.githubusercontent.com/16074363/53136935-276cb600-3581-11e9-874d-9dbb19ec0052.png)
How this is considered reasonable damage is beyond me.

**Comment by Easyfraggy** [comment]
The issue has been open since open beta:

https://github.com/Atlantiss/NetherwingBugtracker/issues/3004

**Comment by Anselm123** [comment]
We did it for the 1st time last night and P2 was literally just RNG if we got through it or not. Flurry attacks seems to have a high proportional ability to always hit as well which seems weird. I have no problem with these mechanics but when tanks get 100-0'd it just makes it shitty to do the boss tbh

**Comment by Anselm123** [comment]
What i mean by always hit btw is that if one of them gets through they all seem to

**Comment by Tereneckla** [comment]
Thrash got removed since
